presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -11,7 +11,7 @@ Synopsis
     SELECT [ ALL | DISTINCT ] select_expr [, ...]
     [ FROM from_item [, ...] ]
     [ WHERE condition ]
-    [ GROUP BY expression [, ...] ]
+    [ GROUP BY [ ALL | DISTINCT ] groupby_expr [, ...] ]
```

Comment:
Let's call this `grouping_element`, to match PostgreSQL: http://www.postgresql.org/docs/9.5/interactive/sql-select.html